index.html

```diff
@@ -565,6 +565,7 @@
 The time immediately after the user agent receives the last byte of the response or immediately before the transport connection is closed, whichever comes first. The resource here can be received either from <a

```

Comment:
@annevk what happens in Fetch if the server closes connection mid-stream (e.g. before all the Content-Length bytes are sent)? Will that trigger "network error" and reject the response promise?